app/models/spotlight/exhibit.rb

```diff
@@ -79,8 +80,8 @@ def solr_data
       end
     end
 
-    def reindex_later
-      Spotlight::ReindexJob.perform_later(self)
+    def reindex_later(user = nil)
```

Comment:
It looks like we attempt to send messages to the user object in the view unchecked, so it seems like the user is required in this case.  Should we explicitly require that a user object is passed?